examples/gno.land/p/demo/dao/vote.gno

```diff
@@ -0,0 +1,70 @@
+package dao
+
+import (
+	"std"
+)
+
+const (
+	VoteAddedEvent = "VoteAdded" // emitted when a vote was cast for a proposal
+
+	VoteAddedIDKey     = "proposal-id"
+	VoteAddedAuthorKey = "author"
+	VoteAddedOptionKey = "option"
+)
+
+type VoteOption string
+
+const (
+	YesVote     VoteOption = "YES"
+	NoVote      VoteOption = "NO"
+	AbstainVote VoteOption = "ABSTAIN"
+)
+
+func (v VoteOption) String() string {
+	return string(v)
+}
+
+// Vote encompasses a single GOVDAO vote
+type Vote struct {
+	Address std.Address // the address of the voter
+	Option  VoteOption  // the voting option
+}
+
+// Stats encompasses the proposal voting stats
+type Stats struct {
+	YayVotes     uint64
+	NayVotes     uint64
+	AbstainVotes uint64
+
+	TotalVotingPower uint64
+}
+
+// YayPercent returns the percentage (0-100) of the yay votes
+// in relation to the total voting power
+func (v Stats) YayPercent() uint64 {
+	return (v.YayVotes / v.TotalVotingPower) * 100
```

Comment:
```suggestion
	return v.YayVotes * 100 / v.TotalVotingPower
```

Reply:
This is probably the biggest nitpick I've resolved 🙃 

[ae650d2](https://github.com/gnolang/gno/pull/2581/commits/ae650d2375181017d3f65762588d349886cfa47e)